pkg/operator/ceph/object/spec.go

```diff
@@ -401,8 +423,17 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 		container.VolumeMounts = append(container.VolumeMounts, mount)
 	}
 	if c.store.Spec.Gateway.CaBundleRef != "" {
-		updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: caBundleExtractedDir, ReadOnly: true}
+		updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: "/etc/ssl/custom-ca", ReadOnly: true}
 		container.VolumeMounts = append(container.VolumeMounts, updatedBundleMount)
+		container.Env = append(container.Env,
+			v1.EnvVar{
+				// When the SSL_CERT_FILE environment variable is set,
+				// OpenSSL-based libraries will use the specified file path
+				// to locate the CA bundle for verifying TLS/SSL certificates
+				Name:  "SSL_CERT_FILE",
```

Comment:
I have some concerns about setting this. Before, on the upstream centos-based ceph images, this wasn't being set, and the `update-ca-trust extract` command was creating several files which might be used by the TLS libs. I worry that setting this will limit the TLS libs to use **only** the specific file, preventing them from picking up other certs.